modules/snooker/src/test/ts/browser/MergeOperationsTest.ts

```diff
@@ -79,4 +79,118 @@ UnitTest.test('MergeOperationsTest', function () {
     ],
     { startRow: 2, startCol: 0, finishRow: 3, finishCol: 0 }
   );
+
+  Assertions.checkMerge(
```

Comment:
All these new tests are about testing the `scope` but it's not mentioned once in the descriptions.

- [ ] Improve test labels/descriptions to better describe what we're testing

Reply:
There also doesn't appear to be a test here for something that ends up generating a `colgroup`, so we need to add that as well.

- [ ] Add test that asserts a colgroup is created as expected

Reply:
The test added for this wasn't a very good one to be testing for this case. It'd be better if this did a column merge not a row merge, as all your new examples are doing a row merge.